### PR TITLE
[QOL-8618] add DCTERMS metadata to dataset and resource pages

### PR DESCRIPTION
**Pull request by ThrawnCA**
- TODO index pages, organisation and group pages

### CODE REVIEW

File: ckanext/publications_qld_theme/templates/organization/read_base.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <meta name="DCTERMS.publisher" scheme="AGLSTERMS.AglsAgent" content="corporateName=The State of Queensland; jurisdiction=Queensland" />
 <meta name="DCTERMS.creator" scheme="AGLSTERMS.GOLD" content="c=AU; o=The State of Queensland; ou={{ group_dict.display_name }}" />
 <meta name="DCTERMS.created" content="{{ h.reformat_date_string(group_dict['created']) }}" />
-<meta name="DCTERMS.description" content="{{ group_dict['description'] or '' }}" />
```
Comment (duttonw) [inline ckanext/publications_qld_theme/templates/organization/read_base.html]: should we not limit description to a number of char's as they can be Huge.

https://www.w3.org/Submission/prism/prism-dc.html#_Toc336961559 

I think we should limit it to 500 char, html escaped string

File: ckanext/publications_qld_theme/templates/package/read.html [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
+<meta name="DCTERMS.publisher" scheme="AGLSTERMS.AglsAgent" content="corporateName=The State of Queensland; jurisdiction=Queensland" />
 {% if pkg['organization'] and pkg['organization']['title'] %}
-  <meta name="DCTERMS.creator" content="c=AU; o=The State of Queensland; ou={{ pkg['organization']['title'] }}" scheme="AGLSTERMS.GOLD" />
+  <meta name="DCTERMS.creator" scheme="AGLSTERMS.GOLD" content="c=AU; o=The State of Queensland; ou={{ pkg['organization']['title'] }}" />
+{% endif %}
+<meta name="DCTERMS.created" content="{{ h.reformat_date_string(pkg['metadata_created']) }}" />
+{% if pkg['metadata_modified'] %}
+  <meta name="DCTERMS.modified" content="{{ h.reformat_date_string(pkg['metadata_modified']) }}" />
 {% endif %}
+<meta name="DCTERMS.description" content="{{ pkg['notes'] or pkg['title'] or pkg['name'] }}" />
```
Comment (duttonw) [inline ckanext/publications_qld_theme/templates/package/read.html:15]: should we html escape description as it can contain html code etc.

**Review by duttonw — APPROVED** [review]

File: ckanext/publications_qld_theme/templates/organization/read_base.html (hunk 1 above)
Comment (ThrawnCA) [reply ckanext/publications_qld_theme/templates/organization/read_base.html]: HTML escaping is built in.

I've added 500-character truncation.

File: ckanext/publications_qld_theme/templates/package/read.html (hunk 2 above)
Comment (ThrawnCA) [reply ckanext/publications_qld_theme/templates/package/read.html]: That's automatic.

**Pull request merged**